src/main/kotlin/com/team4099/robot2023/commands/drivetrain/DrivePathCommand.kt

```diff
@@ -183,6 +211,33 @@ class DrivePathCommand(
   }
 
   override fun initialize() {
+    val containsAllOdometryWaypoints = waypoints.get().all { it is OdometryWaypoint }
+    val containsAllFieldWaypoints = waypoints.get().all { it is FieldWaypoint }
+    Logger.recordOutput("Pathfollow/allRelative", containsAllOdometryWaypoints)
+    Logger.recordOutput("Pathfollow/allField", containsAllFieldWaypoints)
+
+    if ((containsAllOdometryWaypoints && (pathFrame == FrameType.FIELD))) {
+      errorString = "Cannot pass in Relative Waypoints when pathFrame is FrameType.FIELD"
+      end(true)
+    }
+
+    if ((containsAllFieldWaypoints && (pathFrame == FrameType.ODOMETRY))) {
+      errorString = "Cannot pass in Field Waypoints when pathFrame is FrameType.ODOMETRY"
+      end(true)
+    }
+
+    if (!(containsAllFieldWaypoints || containsAllOdometryWaypoints)) {
+      errorString = "Cannot pass in both Field Waypoints and Relative Waypoints"
+      end(true)
+    }
+
```

Comment:
Do we even need `pathFrame` anymore if you're doing this? I feel like the path frame can maybe be implicit and avoid stupid errors

Reply:
i made it explicit so people don't spend a lot of time debugging why the path is scuffed to realize they used a FieldWaypoint instead of an OdometryWaypoint bc they copy pasted another path. it's not tm effort either